### ISSUE TITLE
Bug: Feeds not displaying correctly, some only (Temporarily) work after sorting

### ISSUE THREAD

**Issue by Daviljoe193** [QuiteRSS/quiterss#1382]
Yeah... I can't use this, it's just completely broken. Arch Linux host, QuiteRSS version 0.19.4, though the same issue happens on Windows 10.
![QuiteRSS is dead](https://user-images.githubusercontent.com/67191631/100522148-beaf6600-3176-11eb-8b1a-39e89225b65b.gif)

**Comment by Daviljoe193** [comment]
Labels also doesn't work after sorting, so that's just gone.

**Comment by Daviljoe193** [comment]
Found a way to get it back. Apparently, the database got corrupted, as the below commands were able to get it back.
```
cp ~/.local/share/QuiteRss/QuiteRss/feeds.db .
sqlite3 data.db
.mode insert
.output dump.sql
.dump
.exit

cat dump.sql | grep -v BEGIN_TRANSACTION | grep -v ROLLBACK > dumpn.sql
sqlite3 new.db < dumpn.sql
mv new.db ~/.local/share/QuiteRss/QuiteRss/feeds.db
```
Perhaps @Funcy-dcm can add a feature that detects if the database is corrupted, and if it is, then notify the user that something's wrong, maybe with an option to do something similar to the above commands. It took me until now to even find out it was the database that had issues, and not QuiteRSS, so a feature like what I described would cut down on potentially unnecessary bug reports, like mine.
